assets/scss/6-components/event-block/event-block.html

```diff
@@ -45,15 +47,17 @@ <h3 class="t-uppercase t-lsp-b t-size-xs">Austin | 7 p.m. - 10 p.m.</h3>
   </time>
   <div class="c-event-block__content c-cta-block c-cta-block--no-border l-width-full">
     <div class="c-cta-block__msg t-size-s">
-      <h2 class="c-event-block__title t-serif t-size-l has-xxxs-btm-marg">The Texas Tribune Festival</h2>
-      <h3 class="t-uppercase t-lsp-b t-size-xs">Austin | 7 p.m. - 10 p.m.</h3>
+      <h3 class="has-xxxs-btm-marg t-serif t-links-underlined-hover t-size-m" aria-hidden="true">
+        <a href="#" tabindex="-1">The Texas Tribune Festival</a>
+      </h3>
+      <p class="t-uppercase t-lsp-b t-size-xs">Held in Austin | 7 p.m. - 10 p.m.</p>
```

Comment:
good call on making this a `<p>`. Could we add `t-weight-bold` to this class set to maintain the bold of what's currently portrayed on the site? Are we going to start prefixing "Held in" on the [events page](https://www.texastribune.org/events/)?

Reply:
Added bold. I prepended the "held in" language to make it more clear for screen readers. I think we should eventually bring that everywhere.